packages/webapp/src/components/Forms/NewCropModal/index.js

```diff
@@ -98,6 +98,7 @@ class NewCropModal extends React.Component {
     const currentState = this.state;
     let validated = true;
     let errors = '';
+    let varietyError = '';
```

Comment:
I don't think there is a need to have varietyError, can you use the same structure but use the errors variable? I know the story says the toaster should read "Variety Name is required", but since every other field is reading "not filled out" I would rather be consistent.